fabric/build.gradle

```diff
@@ -13,7 +13,7 @@ dependencies {
     modImplementation "net.fabricmc:fabric-loader:${fabric_loader_version}"
     modImplementation "net.fabricmc.fabric-api:fabric-api:${fabric_version}"
     implementation group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.1'
-    implementation project(":common")
+    modCompileOnly project(":common")
```

Comment:
no need to go through the remapper since the project already has the same mappings.
```gradle
compileOnly project(":common")
```